cli-commands/commands.js

```diff
@@ -428,6 +429,25 @@ const commands = [
 				logger.removeOutputStorage();
 			}
 		}
+	},
+	{
+		command: 'ide [port]',
+		description: 'Runs web-based Solidity IDE that works with the file system',
+		argumentsProcessor: (yargs) => {
+			yargs.positional('port', {
+				describe: 'port to run ganache-cli on',
+				type: 'number'
+			});
```

Comment:
Do we need here to specify default port ?

Reply:
I feel that this port should be not one where a ganache will be ran, but one where ganache has been separately ran by the developer